hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -622,6 +733,35 @@ final Stream<DynamicTest> airdropToNonExistingEvmAddressAccount() {
                             // Any new auto-creation needs to explicitly associate token. So it will be $0.1
                             validateChargedUsd("evmAddressReceiver", 0.1, 1));
         }
+
+        // AIRDROP_19
+        @LeakyHapiTest(overrides = {"entities.unlimitedAutoAssociationsEnabled"})
```

Comment:
I already created a test for AIRDROP-19 [here](https://github.com/hashgraph/hedera-services/pull/14553/commits/d466a5690a17a081c6bdf35468d7e688e6c80942) as the issue in github was not assigned to anyone.